src/data/mv_polynomial.lean

```diff
@@ -1036,6 +1036,32 @@ eval₂_rename_prodmk id _ _ _
 
 end
 
+theorem exists_finset_rename (p : mv_polynomial γ α) :
```

Comment:
Could this have an informative doc-string? The statement is not particularly readable!

Reply:
Maybe this would be easier to understand (and perhaps to use?) if `set.range` wasn't used in the statement, but instead writing the explicit existential for the renamed polynomial? I'm not too concerned either way.

Reply:
Done. Both.